README.md

```diff
@@ -0,0 +1,237 @@
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Rent a Car - Front-end</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [📝 Kanban Board](#kanban-board)
+- [🚀 Live Demo](#live-demo)
+- [🚀 Back-end Link](#back-end)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Rent a Car - Front-end <a name="about-project"></a>
+
+> - Rent a Car frontend is react forntend application for Rent a car. This applications is responsive and users can load, create, and delete cars and reservations. Users need to sign up or sign in to access application. They can sign up or sign in using the sign up and login forms.
+
+## 🛠 Built With <a name="built-with"></a>
+1. React
+2. Redux
+3. CSS
+4. JavaScript
+5. Webpack
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://www.w3.org/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- KANBAN BOARD -->
+
+### KANBAN BOARD
+
+> View the [Kanban board](https://github.com/users/BHObserver/projects/8) for the project here.
+
+> View the initial [screenshot](https://github.com/BHObserver/Rent-a-Car-frontend/issues/2) of the Kanban board here.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+### LIVE DEMO
+
+> Watch [live]() demo of the app or visit the app [page]() to use the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- BACK END -->
+
+### BACK END
+
+> View the [Back-end](https://github.com/BHObserver/Rent-a-Car-backend.git) part of the project here.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy of the project, use this link:
+> 
+```sh
+cd Rent-a-Car-front-end
+https://github.com/BHObserver/Rent-a-Car-frontend
+```
+
+<!-- SETUP -->
+### Setup
+
+To setup this project, run this command:
+
+```sh
+npm start
+```
+### Prerequisites
+
+1. A Code Editor
+2. Git
+
+<!-- INSTALL -->
+### Install
+
+Install this project with Iroko.
```

Comment:
- [ ] Is `Iroko` relevant for this project?  If so, please provide instructions about how to set it up in case it's relevant or needed for the project.

If it's not relevant, kindly remove this line. 👍